packages/eslint-config-airbnb/rules/react.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
   'rules': {
     // Prevent missing displayName in a React component definition
     // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/display-name.md
-    'react/display-name': [0, { 'acceptTranspilerName': false }],
+    'react/display-name': [0, { 'ignoreTranspilerName': false }],
```

Comment:
This is the name of the option, so LGTM. This does happen to invert what it used to be, but since the rule is disabled, it doesn't make much difference if it is `true` or `false`. However, I am in favor of keeping this as `false` as a recommendation.